tests/README.md

```diff
@@ -78,6 +78,15 @@ checks that would typically fail on incomplete stubs (such as `Unknown` checks).
 In typeshed's CI, pyright is run with these configuration settings on a subset of
 the stubs in typeshed (including the standard library).
 
+In order for `pyright_test` to work correctly, some third-party stubs may require dependencies external to typeshed to be installed in your virtual environment prior to running the test.
+You can list or install all of a stubs package's external dependencies using the following script:
+```bash
```

Comment:
_Tiiiny_ nitpick: is `bash` a good syntax highlight to use here? It looks like it might be doing some funky things to the `<` and `>` characters in e.g. `<third_party_stub>`. Maybe no highlighting would be better here?

```suggestion
```
```

Reply:
As far as https://github.com/github/linguist is concerned, `shell`, `sh`, `shell-script`, `bash` and `zsh` are all the same, and look fine.

bash
```bash
(.venv3)$ python tests/get_external_dependencies.py <third_party_stub>  # List external dependencies for <third_party_stub>
(.venv3)$ python tests/get_external_dependencies.py <third_party_stub1> <third_party_stub2>  # List external dependencies for <third_party_stub1> and <third_party_stub2>
(.venv3)$ python tests/get_external_dependencies.py  # List external dependencies for all non stubs packages in typeshed
(.venv3)$ pip install $(python tests/get_external_dependencies.py <third_party_stub>)  # Install non-types dependencies
```

shell
```shell
(.venv3)$ python tests/get_external_dependencies.py <third_party_stub>  # List external dependencies for <third_party_stub>
(.venv3)$ python tests/get_external_dependencies.py <third_party_stub1> <third_party_stub2>  # List external dependencies for <third_party_stub1> and <third_party_stub2>
(.venv3)$ python tests/get_external_dependencies.py  # List external dependencies for all non stubs packages in typeshed
(.venv3)$ pip install $(python tests/get_external_dependencies.py <third_party_stub>)  # Install non-types dependencies
```

powershell
```powershell
(.venv3)$ python tests/get_external_dependencies.py <third_party_stub>  # List external dependencies for <third_party_stub>
(.venv3)$ python tests/get_external_dependencies.py <third_party_stub1> <third_party_stub2>  # List external dependencies for <third_party_stub1> and <third_party_stub2>
(.venv3)$ python tests/get_external_dependencies.py  # List external dependencies for all non stubs packages in typeshed
(.venv3)$ pip install $(python tests/get_external_dependencies.py <third_party_stub>)  # Install non-types dependencies
```

empty
```
(.venv3)$ python tests/get_external_dependencies.py <third_party_stub>  # List external dependencies for <third_party_stub>
(.venv3)$ python tests/get_external_dependencies.py <third_party_stub1> <third_party_stub2>  # List external dependencies for <third_party_stub1> and <third_party_stub2>
(.venv3)$ python tests/get_external_dependencies.py  # List external dependencies for all non stubs packages in typeshed
(.venv3)$ pip install $(python tests/get_external_dependencies.py <third_party_stub>)  # Install non-types dependencies
```

It doesn't outright break the syntax, if anything it highlights that you need to replace this part. It also correctly handles comments and inline commands.

Editors may have different implementations. This is in VSCode with popular markdown syntax highlight (and github-like previewer) that I use:
![image](https://user-images.githubusercontent.com/1350584/210277536-d7a99f33-f61c-4f79-92f0-7676e860f5d1.png)

This is with plugins disabled:
![image](https://user-images.githubusercontent.com/1350584/210277640-a3c80730-1a3d-4a13-b662-cc0f6a4a0cf3.png)

You can see here that "shell" isn't naturally supported in the preview. (code syntax highlight has no difference). But I wouldn't use that as an argument.
Of course the exact look will depend on the editor's implementation.
In any case, having **a** language specifier here only seems better than **no** language specifier.

Another reason to use bash specifically, from a purely semantic point of view, is that the doc seems to assume all commands are to be run on linux, where afaik bash (or bash-like) is most often the default. (and bash-like code usually works in powershell on Windows)

---

I would also like to add a language specifier for better syntax highlight in all relevant code snippets in the docs in a separate PR.

Reply:
> I would also like to add a language specifier for better syntax highlight in all relevant code snippets in the docs in a separate PR.

That sounds like a great idea in general, by the way!